README.md

```diff
@@ -48,38 +53,15 @@ If running in codespaces, this should already be done.
 
 ### AWS Setup
 
-The first thing you'll need to do is configure your aws environment with:
-
-```bash
-aws configure
-```
-
-Enter in your AWS Access Key ID and AWS Secret Access Key, which can be obtained by setting up a user account in the AWS console. For region, set it to the AWS region you'd like to set up your SDS. For IMAP, we're using "us-west-2"
-
-If you have multiple AWS access/secret key pairs locally, you can add the configuration to `~/.aws/config`. 
-
-```
-[imap]
-region=us-west-2
-aws_access_key_id=<Access Key>
-aws_secret_access_key=<Secret Key>
-```
-
-Then, you can set the profile used by cdk by setting the `AWS_PROFILE` environment variable to the profile name (in this case, imap):
-
-```
-export AWS_PROFILE=imap
-```
+[AWS Setup page](docs/source/cdk/aws-setup.rst)
```

Comment:
```suggestion
[AWS Setup page](https://sds-data-manager.readthedocs.io/en/latest/cdk/aws-setup.html)
```
I think @maxinelasp was wanting to link out to our built HTML docs instead of the rst which isn't rendered all that nicely on GitHub.

Reply:
Yes, links should always go to the rendered version in readthedocs. Inside the rst files you can use either [ref](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-ref-role) or [doc](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role) roles.